src-vue/src/views/DeveloperView.vue

```diff
@@ -207,7 +186,26 @@ export default defineComponent({
             await persistentStore.clear();
             // ...and save
             await persistentStore.save();
-        }
+        },
+        async createNewWindow() {
+            await invoke("open_repair_window").then((message) => {
```

Comment:
I don't think we should show notifications here.

Reply:
Removed in 50bf106284f28211b4f268153c8201f467104d34